src/renderer/handle-events.js

```diff
@@ -91,6 +91,14 @@ export const handleEvents = () => {
     store.dispatch(updateServerAttrib({ AUGUR_BULK_SYNCING: false }))
   })
 
+  ipcRenderer.on(BULK_ORPHANS_CHECK_STARTED, () => {
+    store.dispatch(updateServerAttrib({ AUGUR_BULK_ORPHANS_CHECK: true }))
```

Comment:
Is AUGUR_BULK_ORPHANS_CHECK or AUGUR_BULK_ORPHANS_CHECK  attribute needed in the UI, I don't see it used. if not no reason to save to redux

Reply:
Right now we do this for BULK_SYNC events too. My assumption is its nice to have the plumbing in there for when we do want to surface something in the UI

Reply:
works for me.